ghscraper.py

```diff
@@ -253,21 +253,26 @@ def scrapePullRequestComments(repo, repoPath):
                       datetime.datetime.now() + datetime.timedelta(minutes = GITHUB_RATELIMIT_INTERVAL))
                 time.sleep(60*GITHUB_RATELIMIT_INTERVAL)
 
+def __login(args):
```

Comment:
Question on this, since I'm less familiar with Python style guides: Why two underscores? From PEP8, it seems like one underscore is enough for the function to be excluded from module import, and two underscores are normally used for class attributes. I'm curious as to your reasoning on this, but otherwise the changes look great! Thanks for doing this. :heartbeat: